README.md

```diff
@@ -1,2 +1,67 @@
-# heatmap
+# Heatmap
 Fifth project on Microverse course
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Making a Heat Map out of a page.
+
+![screenshot](./src/images/app_screenshot.png)
+
+We described how would look [this](https://www.smashingmagazine.com/) page as if using a Heat Map.
+
+The darker the color is, the most likely is to get visited by a user.
+
+
+
+## Built With
+
+- HTML:
+- - Using semantic tags like: main, article, footer.
+- CSS:
+- - Using styling for Grids, Grayscale for most visited parts of the page.
+- FONTAWESOME:
+- - As a source of Icons.
+
+## Live Demo
+
+Here you will find a Live Demo page, any comment is well recieved.
+
+[Live Demo Link](https://raw.githack.com/MrkarlosM/heatmap/carlos-branch/index.html)
+
+## Authors
+
+Please feel free to contact us in any time:
+
+👤 **Carlos Mario **
+
+- GitHub: [@githubhandle](https://github.com/MrkarlosM)
+- Twitter: [@twitterhandle](@MrkarlosM)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/carlos-mario-martinez-b1768355/)
```

Comment:
You should replace @githubhandle, @twitterhandle and Linkedin texts in your links with your respective usernames you use on those sites